### DISCUSSION TITLE
Stata and GitHub Integration

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34746] [New to GitHub] [answered]
Hello,

I am new to GitHub and would like to know if it is possible to sync my project files directly from Stata to GitHub. 

Thanks in advance!

**Comment by jfullstackdev** [comment, marked as answer]
I think this is the official documentation 

https://www.stata.com/stata-news/news36-3/community-corner-github/

**Reply by ghost** [reply]
Thank you!
